pkg/sync/sync.go

```diff
@@ -85,22 +84,34 @@ func SyncUsers(ctx context.Context, clientService *admin.Service, cfg *config.Co
 		if usersToCreate != nil {
 			log.Println("✎ Creating...")
 			for _, user := range usersToCreate {
-				glib.CreateUser(*clientService, &user)
-				log.Printf("\t+ user: %s\n", user.PrimaryEmail)
+				err := glib.CreateUser(*clientService, &user)
+				if err != nil {
+					return fmt.Errorf("⚠ Failed to create user %s: %v.", user.PrimaryEmail, err)
+				} else {
+					log.Printf("\t+ user: %s\n", user.PrimaryEmail)
+				}
 			}
 		}
 		if usersToDelete != nil {
 			log.Println("✁ Deleting...")
 			for _, user := range usersToDelete {
-				glib.DeleteUser(*clientService, user)
-				log.Printf("\t- user: %s\n", user.PrimaryEmail)
+				err := glib.DeleteUser(*clientService, user)
+				if err != nil {
+					return fmt.Errorf("⚠ Failed to delete user %s: %v.", user.PrimaryEmail, err)
+				} else {
+					log.Printf("\t- user: %s\n", user.PrimaryEmail)
```

Comment:
I would discourage tabs here, if you aligned your text to match the emoji-width (for the warning sign icons and stuff). If you just want a random indentation it's fine, otherwise my tabwidth is most likely not your tabwidth and the output can look weird for me.